### PR TITLE
Add circleci

### PR DESCRIPTION
**Pull request by zachharris1**
# Description of change
adds a circleci config to run tests

# Manual QA steps
 - 
 
# Risks
 - 
 
# Rollback steps
 - revert this branch

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+          command: |
+            source /usr/local/share/virtualenvs/tap-crossbeam/bin/activate
+            pylint tap_crossbeam --disable 'broad-except,chained-comparison,empty-docstring,fixme,invalid-name,line-too-long,missing-class-docstring,missing-function-docstring,missing-module-docstring,no-else-raise,no-else-return,too-few-public-methods,too-many-arguments,too-many-branches,too-many-lines,too-many-locals,ungrouped-imports,wrong-spelling-in-comment,wrong-spelling-in-docstring,bad-whitespace'
+      - slack/status:
+          channel: 'stitch-tap-tester-tests'
+          mentions: "${CIRCLE_USERNAME}"
+          fail_only: true
+          only_for_branches: master
+          failure_message: |
+            :red_circle: ${CIRCLE_JOB} failed!
```
Comment (luandy64) [inline .circleci/config.yml:34]: This seems strange to break from our usual `slack/notify-on-failure`

File: tap_crossbeam/client.py [hunk 2]
```diff
@@ -53,13 +48,10 @@ def refresh_access_token(self):
...
         self.__access_token = data['access_token']
 
-        self.__user_expires_at = datetime.utcnow() + \
-            timedelta(seconds=data['expires_in'] - 10) # pad by 10 seconds for clock drift
-
     @backoff.on_exception(backoff.expo,
                           (Server5xxError,
                            RateLimitException,
-                           ConnectionError,
+                           requests.exceptions.ConnectionError,
```
Comment (luandy64) [inline tap_crossbeam/client.py:54]: Is this a pylint thing? It seemed fine to me before

**Review by luandy64 — APPROVED** [review]
I have questions, but this looks good to me

File: tap_crossbeam/client.py (hunk 2 above)
Comment (zachharris1) [reply tap_crossbeam/client.py:54]: This was something I did with Dan involving two different ConnectionError classes. `requests.exceptions.ConnectionError` is different from just `ConnectionError`

File: .circleci/config.yml (hunk 1 above)
Comment (zachharris1) [reply .circleci/config.yml:34]: This is just cut and paste from how-to-sources, which might be out of date

**Pull request merged**